metriken/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "metriken"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
You don't need to do this. Cargo will automatically choose the newer version of histogram.

Reply:
reverted the version bump